test/blocks/marquee/mocks/video.html

```diff
@@ -23,7 +23,7 @@ <h2 id="heading-xl-marquee-standard-medium-left">Heading XL Marquee standard med
 <div class="marquee" id="multiple-background">
   <div>
     <div><a href="https://business.adobe.com/content/dam/dx/us/en/product-tours/workfront/video/Chapter%201_Video01.mp4">https://business.adobe.com/content/dam/dx/us/en/product-tours/workfront/video/Chapter%201_Video01.mp4</a></div>
-    <div><a href="/drafts/methomas/media_16e79d59a997bc94e5f1b2a518acb152a14f97490.mp4">https://main--milo--adobecom.hlx.page/media_16e79d59a997bc94e5f1b2a518acb152a14f97490.mp4</a></div>
+    <div><a href="https://main--milo--adobecom.hlx.page/media_16e79d59a997bc94e5f1b2a518acb152a14f97490.mp4">https://main--milo--adobecom.hlx.page/media_16e79d59a997bc94e5f1b2a518acb152a14f97490.mp4</a></div>
```

Comment:
test files should not have external resources as dependencies.

Reply:
Unfortunately, web test runner is not running base of hlx localhost which means, all relative URLs pointing our content base are getting 404. 
(See this example https://github.com/adobecom/milo/actions/runs/6540291246/job/17759947301?pr=1343#step:6:1491)
This is sometimes frustrating when you need to debug your test deeply. There are many other test mockup body using external URLs. 

If you think this is really bugging you, I can still undo my test URLs, I don't think it will really affect existing coverage at this point but adding little more 404s on test.

Reply:
actually I'm probably wrong. What I said is probably for js/css files, so it should be ok.